pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -646,9 +653,7 @@ func (ssc *defaultStatefulSetControl) rollingUpdateStatefulsetPods(
 		if utilfeature.DefaultFeatureGate.Enabled(features.StatefulSetAutoResizePVCGate) &&
 			set.Spec.VolumeClaimUpdateStrategy.Type == appsv1beta1.OnPodRollingUpdateVolumeClaimUpdateStrategyType {
 			// resize pvc if necessary and wait for resize completed
-			if match, err := ssc.podControl.IsClaimsCompatible(set, replicas[target]); err != nil {
-				return status, err
-			} else if !match {
+			if !pvcMatched {
```

Comment:
in L 646,  shall we also log the volumes to be resizing? otherwise we can not tell whether the rolling is blocked by unavailable pods or resizing volumes.

Reply:
Now resizing volumes lead to pods becoming unavailable. To address this, I have added logs at lines 621 and 622. 

In the future, when we add parallelism in the volumeUpdateStrategy, it will enable users to control the maximum number of volumes being updated simultaneously, thereby separately accounting for these two causes of unavailability during updates.